JobConfig/primary/RMCInternalEndpoint.fcl

```diff
@@ -0,0 +1,8 @@
+#
+# RMC Internal, kmax = 101.9 MeV
+#
+
+#include "Production/JobConfig/primary/RMCInternal.fcl"
+physics.producers.generate.decayProducts.spectrum.ehi      : 101.9
```

Comment:
more cases of repeated numbers and config.
If nesting doesn’t work could the value at least be set in prolog?

Reply:
I moved this to the prolog.